README.md

```diff
@@ -212,13 +208,12 @@ We welcome community contributions to nGraph. If you have an idea for how to
 improve it:
 
 * Share your proposal via [GitHub issues].
-* Make sure your patch is in line with Google style by setting up your git `pre-commit` hooks.  First, ensure `clang-format` is in your path, then:
-
-   ```
-   pip install pre-commit autopep8 pylint
-   pre-commit install
-   ```
-
+* Make sure your patch is in line with Google style by setting up your git 
+  `pre-commit` hooks. First, ensure `clang-format` is in your path, then:
+   
+        pip install pre-commit autopep8 pylint
+        pre-commit install
```

Comment:
This instruction is old. @avijit-nervana do we ask contributors to have clang-format-3.9, yapf0.24+py2 and just run main/apply... ?

Reply:
removed this since it is TF-specific and can be handled on their end.